driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -621,8 +634,7 @@ public interface StateListener {
   public interface LifecycleAwareStateListener extends StateListener {
     /**
      * Gets invoked when the listener is registered with a cluster, or at cluster startup if the
-     * listener was registered at initialization with {@link
-     * com.datastax.driver.core.Cluster#register(Host.StateListener)}.
```

Comment:
Please move these kind of changes into a separate commit with proper message to it that explains why it is done